test/support/rails/railtie.rb

```diff
@@ -0,0 +1,15 @@
+module Rails
+  class Railtie
+    class Initializer < Struct.new(:name, :block)
+      def run
+        block.call
+      end
+    end
+
+    class_attribute :initializers, default: []
```

Comment:
FYI the `default` option was added on 5.2 only. The gem dependency requires `activesupport >= 4`.

Let's initialize the initializers (lol) variable manually

Reply:
I was wondering why it wasn't working. Thanks!